docs/conf.py

```diff
@@ -67,6 +67,10 @@
 
 autoclass_content = "both"
 autodoc_preserve_defaults = True
+autodoc_default_options = {
+    "members": True,
+    "undoc-members": True,
+}
```

Comment:
I've removed these default options from the other containers and the Widget base class: we can clean up the others in a separate PR.